setup.py

```diff
@@ -53,17 +53,18 @@ def build_extension(self, ext: CMakeExtension):
         extdir = str(Path(self.get_ext_fullpath(ext.name)).parent.absolute())
 
         debug = int(os.environ.get("DEBUG", 0)) if self.debug is None else self.debug
-        cfg = "Debug" if debug else "Release"
         ninja_path = str(shutil.which('ninja'))
 
         # Set Python_EXECUTABLE instead if you use PYBIND11_FINDPYTHON
         configure_args = [
             "-GNinja",
             f"-DCMAKE_LIBRARY_OUTPUT_DIRECTORY={extdir}",
             f"-DPYTHON_EXECUTABLE={sys.executable}",
-            f"-DCMAKE_BUILD_TYPE={cfg}",  # not used on MSVC, but no harm
             f"-DCMAKE_MAKE_PROGRAM={ninja_path}",
         ]
+
+        if debug:
+            configure_args += ["-DCMAKE_BUILD_TYPE=Debug"]
```

Comment:
with this new setup, where `-DCMAKE_BUILD_TYPE` is set to `Release` otherwise?

Reply:
If `--debug` is not given, the default one set in `CMakeLists.txt` is used. Currently, it is `RelWithDbgInfo` (see #201).